src/components/CookieWarning.js

```diff
@@ -36,21 +33,28 @@ export const CookieWarning = () => {
     }, cookies)
     signOut()
   }
-  return !(acceptedCookies) && div({
+  return !cookiesAccepted && div({
     style: {
       position: 'fixed', height: 100, bottom: 0, width: '100%',
       display: 'flex', alignItems: 'center', justifyContent: 'space-between',
-      backgroundColor: 'white', borderTop: `6px solid ${colors.primary()}`
+      backgroundColor: colors.dark(0.15), borderTop: `6px solid ${colors.primary()}`
     }
   }, [
-    div({ style: { padding: '2rem' } }, ['Terra uses cookies to enable the proper functioning and security of our website,',
-      ' and to improve your experience on our site. By clicking Agree or continuing to use our site, you consent to the use of these functional',
-      ' cookies. If you do not wish to allow use of these cookies, you may tell us that by clicking on Reject. As a result, you will be unable',
-      ' to use our site. To find out more, read our ',
-      h(Link, { style: { textDecoration: 'underline' }, href: Nav.getLink('privacy') }, ['privacy policy']), '.']),
+    div({ style: { padding: '0.9rem 2rem', height: '100%', display: 'flex', alignItems: 'center' } }, [
+      div({ style: { overflowY: 'auto', height: '100%' } }, [
+        `${getAppName()} uses cookies to enable the proper functioning and security of our website,
+        and to improve your experience. By clicking Agree or continuing to use our site, you consent to the use of these functional
+        cookies. If you do not wish to allow use of these cookies, you may tell us that by clicking on Reject. As a result, you will be unable
+        to use our site. To find out more, read our `,
+        h(Link, { style: { textDecoration: 'underline' }, href: Nav.getLink('privacy') }, ['privacy policy']), '.'
+      ])
+
+    ]),
     div({ style: { padding: '2rem', display: 'flex' } }, [
       h(ButtonPrimary, { onClick: () => acceptCookies(true) }, ['Agree']),
       h(ButtonSecondary, { style: { marginLeft: '2rem' }, onClick: () => { rejectCookies() } }, ['Reject'])
     ])
   ])
 }
+
+export default CookieWarning
```

Comment:
just out of curiosity, why is it better to `export default` here instead of inline? (also noticed it was changed for `SignIn`). Is it easier to find the exported value when we specify it at the end of the file?
I see that `export default` is used when there is just 1 value so I think I understand that part at least.

Reply:
See https://github.com/DataBiosphere/terra-ui/pull/2365#discussion_r567131664

Reply:
interesting, thanks!